python/ray/serve/_private/proxy_state.py

```diff
@@ -46,39 +39,40 @@ def actor_id(self) -> str:
         raise NotImplementedError
 
     @abstractmethod
-    def start_new_ready_check(self):
-        """Start a new ready check on the proxy actor."""
-        raise NotImplementedError
-
-    @abstractmethod
-    def start_new_health_check(self):
-        """Start a new health check on the proxy actor."""
-        raise NotImplementedError
-
-    @abstractmethod
-    def start_new_drained_check(self):
-        """Start a new drained check on the proxy actor.
+    def is_ready(self, timeout_s: float) -> Optional[bool]:
+        """Return the payload from proxy ready check when ready.
```

Comment:
What is the "payload" here? Isn't it just returning a bool indicator?

Reply:
Yeah, need to fix that